technologies/job/r/r-3.4.4-base/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM rocker/tidyverse:3.4.4
+
+USER root
```

Comment:
l'idéal sera de lancer ce docker avec un autre user que root ... `rstudio` peut etre ...

Reply:
but it won't be iso with previous version "pre migration"... (one change at a time - moreover the R image doesn't provide a specific user afaik)